doc/nrf/config_and_build.rst

```diff
@@ -24,7 +24,7 @@ All of them have a role in the creation of an application, from configuring the
 Read the guides in this section to learn how to use these tools and configuration methods in areas valid for :ref:`all supported boards <app_boards>`.
 Depending on the board you are working with, check also its :ref:`hardware guide <device_guides>`, as some nRF Series families have specific exceptions and rules to be applied on top of the general configuration procedures.
 
```

Comment:
I think we should add a sentence explaining the differences and similarities of Bootloaders and DFU here.
Not sure if this goes here or in the bootloader or DFU page.
Maybe:
```suggestion
"A bootloader and Device Firmware Updates (DFU) are closely related, but essentially are different functionalities. The bootloader is a firmware image, and has two main jobs. First, it verifies the next firmware image, such as the application. Secondly, the bootloader can choose which firmware image to boot next, which is useful when updating the application as part of DFU. DFU is a procedure in two steps. First a new firmware image is transferred to the chip, and then the bootloader tests and boots the new firmware. 
As such, bootloaders and DFU has separate chapters: ref:`app_bootloaders` and :ref:`app_dfu`.
```

Reply:
This I think makes more sense on the intro pages in each section.